src/components/Home.js

```diff
@@ -0,0 +1,134 @@
+
+import React from 'react'
+import styled from 'styled-components';
+import banners from '../data/featured-banners.json'
+import productCategories from '../data/product-categories.json'
+import FeaturedProducts from './FeaturedProducts';
+
+
+const DivCategory = styled.div`
+        border: 1px solid;
+        min-width: 30vh;
+        background-color: #5472d3;
+    `;
+
+const ImageInSlider = styled.img`
+        max-height: 100vh;
+        width: 100vh;
+        `;
+
+
+function Categories(){
+    const categories = productCategories.results.map((obj) => {
+        return obj.data.name
+    });
+    const [categoryItems,setCategoryitems] = React.useState(categories.slice(0,3));
+    const [section,setSection] = React.useState(0);
+    
+
+    
+
+    const Category = ({name}) => {
+        return (
+        <DivCategory key={name} className="category">
+            <p>{name}</p>
+        </DivCategory>);
+    }
+
+    function Elements(props) {
+        const items = props.items;
+        const elements = items.map((item) => {
+            return <Category key={item} name={item}/>
+        });
+
+        return (
+            elements
+        );
+    }
+    
+
+    const handleChangeNext = (event) => {
+        if(section + 1 < 3){
+            const columns = [];
+            var tmp = section +1
```

Comment:
Remember that var in js is for global variables, in this case is better use let to define a tmp variable